docs/tutorial.md

```diff
@@ -31,6 +31,18 @@ Here's what the finished app will look like:
 ```{.textual path="docs/examples/tutorial/stopwatch.py" title="stopwatch.py" press="tab,enter,tab,enter,tab,enter,tab,enter"}
 ```
 
+### Try it out!
+
+The following is *not* a screenshot, but a fully interactive Textual app running in your browser.
+
+
+!!! textualize "Try in Textual-web"
```

Comment:
You are using `!!!` here and `???` in the other examples. Is this OK?

Reply:
The `???` puts it in an expandable container. For the tutorial I wanted it to already be expanded.

Reply:
Understood.